app/react/reducers/index.js

```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from 'redux';
+
+import MessagesReducer from './reducer_messages';
+import ActiveRoomReducer from './reducer_active_room';
+import RoomsReducer from './reducer_rooms';
```

Comment:
I think you don't need to prepend `reducer_` to each filename here. it's already in `reducers` directory, so we know these are reducers